app/views/registrations/show.html.erb

```diff
@@ -1,2 +1,58 @@
-<h1>Thanks for registering!</h1>
-<p>Your registration number is <strong><%= @voter['registration_number'] %></strong></p>
\ No newline at end of file
+<!DOCTYPE html>
```

Comment:
You shouldn't need the full html wrapper here.  The view will be embedded in the layout.